api/db/populate-attendants.js

```diff
@@ -7,12 +11,21 @@ const Attendant = require('../models/Attendant');
 let displayLogs;
 
 const createAttendant = async (attendantParams) => {
-  const attendant = new Attendant({
-    event: attendantParams.event,
-    user: attendantParams.user,
-  });
+  const attendant = new Attendant(attendantParams);
   await attendant.save();
 
+  // New attendee notification
+  await createNotification(
```

Comment:
This notification has to be created in all cases but when the referenced user is the event creator